packages/types/index.d.ts

```diff
@@ -1 +1,63 @@
 import * as core from "./lib/core";
+
+export interface Script {
+  code_hash: string;
+  hash_type: string;
+  args: string;
+}
+
+export interface OutPoint {
+  tx_hash: string;
+  index: string;
+}
+
+export interface CellDep {
+  out_point: OutPoint;
+  dep_type: string;
+}
+
+export interface Cell {
+  cell_output: {
+    capacity: string;
+    lock: Script;
+    type?: Script;
+  };
+  data: string;
+  out_point: OutPoint;
+  block_hash: string;
+  block_number: string;
+}
+
+export interface QueryOptions {
+  lock?: Script | null;
```

Comment:
I think we can just allow `Script` or `undefined`, and reject `null`

Reply:
updated